src/i18n/en.json

```diff
@@ -1782,7 +1783,9 @@
       "terms-title": "Terms",
       "download-terms-btn": "Download terms",
       "already-in-your-balance-btn": "Already in your balances",
-      "owner-lbl": "Owner"
+      "owner-lbl": "Owner",
+      "balances": "Balances",
```

Comment:
`balances-subheading`. add roles to make life easier for translators (if we get some one day)